addon_service/credentials/models.py

```diff
@@ -99,16 +109,20 @@ def authorized_accounts(self):
                 *filter(
```

Comment:
Minor: docstring for this function could be updated, as it's not longer accurate